config/gcp.spc

```diff
@@ -20,6 +20,9 @@ connection "gcp" {
   # If not set, no impersonation is done.
   #impersonate_service_account = "YOUR_SERVICE_ACCOUNT"
 
+  # `ignore_error_messages` (optional) - List of additional GCP error message pattern to ignore for all queries.
+  #  ignore_error_messages = ["^.*API has not been used.*$"]
```

Comment:
What error does a user receive if an invalid regex is passed in?

Also, do users need to use `^` and `$`, or will we still match without those explicitly included in a pattern?

Reply:
The user will receive an error `error parsing regexp: invalid escape sequence:` during the query execution if a user provide an invalid regex.

User may include `^` and `$` or may not be, in both of the case the code changes are working fine.